src/appleseed.studio/mainwindow/project/attributeeditor.cpp

```diff
@@ -78,10 +81,13 @@ void AttributeEditor::edit(
     if (value_provider)
         m_value_provider = value_provider;
 
+    renderer::ParamArray param_array;
+
     m_entity_editor.reset(
         new EntityEditor(
             m_parent,
             m_project,
+            param_array,
```

Comment:
That won't work, you need to pass settings here.

Reply:
Now I understand what setting_key does!